packages/cli/src/auth/jwt.ts

```diff
@@ -38,7 +38,6 @@ export function issueJWT(user: User): JwtToken {
 
 	const signedToken = jwt.sign(payload, config.getEnv('userManagement.jwtSecret'), {
 		expiresIn: expiresIn / 1000 /* in seconds */,
-		algorithm: 'HS256',
```

Comment:
[This was explicitly added to keep auth tokens backward compatible](https://github.com/n8n-io/n8n/pull/5116/files#diff-a7bbc94f4a504762cbb20b2f7e0abd95c52cef49925dd726ce7576997c2f5c1aR30). 
Any specific reason for removing this?

Reply:
Because that is the default's algorithm. 

<img width="334" alt="image" src="https://github.com/n8n-io/n8n/assets/16496553/13f2cb63-b2a4-455e-8b4d-64b166b5c6ad">